pkgs/misc/sagetex/default.nix

```diff
@@ -15,8 +16,14 @@ stdenv.mkDerivation (finalAttrs: rec {
     sha256 = "sha256-OfhbXHbGI+DaDHqZCOGiSHJPHjGuT7ZqSEjKweloW38=";
   };
 
-  buildInputs = [
+  outputs = [ "tex" ];
+
+  nativeBuildInputs = [
     texlive.combined.scheme-basic
+    # multiple-outputs.sh fails if $out is not defined
+    (writeShellScript "force-tex-output.sh" ''
+      out="''${tex-}"
+    '')
```

Comment:
This is the ugly bit: it's tricky to generate a TeX package with only output `tex`. We should offer a generic hook `texlive.forceTeXOutput`... ?